rust/wasm/compile/default.nix

```diff
@@ -14,7 +14,7 @@ let
  ''
  set -euxo pipefail
  export WASM_PATH=${path}/
- cargo build --release --target wasm32-unknown-unknown --manifest-path "$WASM_PATH"Cargo.toml --target-dir "$HC_TARGET_PREFIX"/"$WASM_PATH"target;
+ cargo build --release --target wasm32-unknown-unknown --manifest-path "$WASM_PATH"Cargo.toml --target-dir "''${HC_TARGET_PREFIX:-.}"/"$WASM_PATH"target;
```

Comment:
what the hey is this? how does adding a kodama emoticon `:-.` help?

Reply:
haha : ) - `:-` is bash variable expansion default value, so `${VAR:-DEF}` will result in `DEF` if `VAR` is not set.